instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaPropagation.java

```diff
@@ -0,0 +1,33 @@
+package brave.kafka.clients;

```

Comment:
add file package-info.java with the following:
```java
@javax.annotation.ParametersAreNonnullByDefault
package brave.kafka.clients;
```

Reply:
Already done : https://github.com/ImFlog/brave/blob/5d02da3a5e010d5ed6b161e8abbcd590ca52f77f/instrumentation/kafka-clients/src/main/java/brave/kafka/clients/package-info.java